examples/flutter_gallery/lib/demo/progress_indicator_demo.dart

```diff
@@ -63,43 +63,51 @@ class _ProgressIndicatorDemoState extends State<ProgressIndicatorDemo> with Sing
 
   Widget _buildIndicators(BuildContext context, Widget child) {
     List<Widget> indicators = <Widget>[
-        new SizedBox(
-          width: 200.0,
-          child: new LinearProgressIndicator()
-        ),
-        new LinearProgressIndicator(),
-        new LinearProgressIndicator(),
-        new LinearProgressIndicator(value: _animation.value),
-        new CircularProgressIndicator(),
-        new SizedBox(
-            width: 20.0,
-            height: 20.0,
-            child: new CircularProgressIndicator(value: _animation.value)
-        ),
-        new Text('${(_animation.value * 100.0).toStringAsFixed(1)}%${ _controller.isAnimating ? "" : " (paused)" }')
+      new SizedBox(
+        width: 200.0,
+        child: new LinearProgressIndicator()
+      ),
+      new LinearProgressIndicator(),
+      new LinearProgressIndicator(),
+      new LinearProgressIndicator(value: _animation.value),
+      new Row(
+        mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+        children: <Widget>[
+          new CircularProgressIndicator(),
+          new SizedBox(
+              width: 20.0,
+              height: 20.0,
+              child: new CircularProgressIndicator(value: _animation.value)
+          ),
+          new Text('${(_animation.value * 100.0).toStringAsFixed(1)}%${ _controller.isAnimating ? "" : " (paused)" }'),
```

Comment:
Laying them out in a row like this causes the animation to jank a bit because of the changing size of the `Text`.